orc8r/cloud/go/services/bootstrapper/servicers/registration/cloud_registration.go

```diff
@@ -0,0 +1,154 @@
+package registration
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"magma/orc8r/lib/go/protos"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	rootCAFilePath = "/var/opt/magma/certs/rootCA.pem"
```

Comment:
Can I do this on init rather than it being passed in?

Reply:
Inits are generally a bad idea as they rely on global state rather than component-level state, and e.g. make testing much more difficult. I'd opt for passing in here via explicit dependency injection

Reply:
Isn't `rootCA` a global state though? I'll do this but just wondering